File: TsvToCsv/force.py
import pandas as pd
import os
import sys
# subject01_walk1_Actuation_force.tsv
# vas_int_r

args = sys.argv
IFILE = str(args[1])
COL = str(args[2])

tsv = pd.read_table(IFILE, header=22, usecols=[COL])
df = pd.DataFrame(tsv)
df.T.to_csv( '_export/'+COL+'.csv', header=False, index=False)

File: TsvToCsv/force.cxx
#include <iostream>

int main(int argc, char *argv[])
{
    double array[]=
    {
        #include "_export/vas_int_r.csv"
    };

    int size = sizeof(array) / sizeof(double);
    for (int i = 0; i < size; i++)
    {
        std::cout << array[i] <<std::endl;
    }
    
    return 0;
}